apiserver/observer/auditfilter.go

```diff
@@ -105,12 +105,17 @@ func (l *bufferedLog) flush() error {
 	return nil
 }
 
-// InterestingRequest returns whether this API request is interesting enough
-// to write the conversation to the audit log.
-func InterestingRequest(req auditlog.Request) bool {
-	return !readOnlyMethods.Contains(fmt.Sprintf("%s.%s", req.Facade, req.Method))
+// MakeExclusionFilter returns a filter function for audit logging
+// that will return false if the request's method matches any of the
```

Comment:
Can we reword this comment to define how a match (true) is obtained. It's more readable to explain how the match criteria for a filter works I think.

The "MakeExclusionFilter" name is then also confusing. Isn't this replacing "InterestingRequest". So we really are wanting to make a filter what returns true if the request is interesting. So why not name it "MakeInterestingRequestFilter". The internal implementation of !excludeMethods.Contains() says the same. This name also allows future extensions like "audit *only* these methods".

Reply:
Good call - I've renamed it to `MakeInterestingRequestFilter` and reworded the comment.